README.md

```diff
@@ -1,30 +1,46 @@
 # Vesktop
 
-Vesktop is a custom Discord desktop app
+<img align="right" width="64" src="static/shiggy.gif">
+<img align="right" width="64" src="https://avatars.githubusercontent.com/u/113042587">
 
-**Main features**:
-- Vencord preinstalled
-- Much more lightweight and faster than the official Discord app
-- Linux Screenshare with sound & wayland
-- Much better privacy, since Discord has no access to your system
+Vesktop is a custom Discord Desktop app.
 
-**Not yet supported**:
-- Global Keybinds
-- see the [Roadmap](https://github.com/Vencord/Vesktop/issues/324)
+## Main features
 
-![](https://github.com/Vencord/Vesktop/assets/45497981/8608a899-96a9-4027-9725-2cb02ba189fd)
-![](https://github.com/Vencord/Vesktop/assets/45497981/8701e5de-52c4-4346-a990-719cb971642e)
+- Vencord pre-installed.
+- Much more lightweight and faster than the official Discord app.
+- Linux screenshare with sound and Wayland compatibility.
+- Much better privacy, since Discord has no access to your system.
+- Fixes scroll issues and other issues/nuisances present on the official Discord app for Linux.
+
+### Not yet supported
+
+- Global keybinds.
+- For other things, see the [roadmap](https://github.com/Vencord/Vesktop/issues/324).
+
+## Screenshots
+
+<table>
+<tr>
+<th colspan="3">Linux Screenshare</th>
```

Comment:
focuses a bit too much on linux... there should also be one image that just shows normal discord